Metalama.Framework.DesignTime.Contracts/Preview/ITransformationPreviewService.cs

```diff
@@ -24,4 +24,23 @@ public interface ITransformationPreviewService : ICompilerService
             IPreviewTransformationResult[] result,
             CancellationToken cancellationToken );
     }
+
+    /// <summary>
+    /// Defines a method that allows to see the generated code for a single file.
+    /// This service is used to produce the generated code for introduced types.
+    /// </summary>
+    [ComImport]
+    [Guid( "2D800D48-3BF1-4EF8-98F5-62FA4417F3F7" )]
+    public interface ITransformationPreviewService2 : ITransformationPreviewService
+    {
+        /// <summary>
+        /// Transforms a single file.
+        /// </summary>
+        Task PreviewGeneratedFileAsync(
+            Project project,
+            string filePath,
```

Comment:
Also maybe a single `filePaths` parameter would be preferable (if not, document.)